lib/conf.c

```diff
@@ -0,0 +1,383 @@
+/*
+ * Copyright (c) 2011-2015 Juan Romero Pardines.
```

Comment:
Why not just convert to SPDX for this file and future files?

Reply:
You need to copy these sorts of licenses with the source.

Reply:
I would still suggest adding SPDX to make processing easier.